source/shared/support/_sidebar.html.erb

```diff
@@ -9,6 +9,9 @@
     <li class="sub-navigation__item">
       <a href="/support/connect-visitors-to-govwifi">Connect visitors to GovWifi</a>
     </li>
+    <li class="sub-navigation__item ">
+      <a href="#">Check your organisation email address</a>
```

Comment:
Maybe change this to be the current page's URL rather than `#`?

Reply:
# is a general convention I've always used when developling webpages that don't link anywhere yet.

Reply:
That's fine. 
Was thinking just for consistency since all the other links are their own URL.